server/athenian/api/openapi/openapi.yaml

```diff
@@ -838,6 +838,219 @@ paths:
 # {% endif %}
       x-openapi-router-controller: athenian.api.controllers.reposet_controller
       x-codegen-request-body-name: body
+  /team/create:
+    post:
+      operationId: create_team
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/TeamCreateRequest'
+        x-body-name: body
+      responses:
+        200:
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CreatedIdentifier'
```

Comment:
yup totally makes sense to me, I just almost blindly copied what was being sent during the `PUT`, gonna change it since all those info are available to the api 👍

Reply:
done [here](https://github.com/athenianco/athenian-api/pull/334/commits/b197c83d65ad261309a3cdb0c546443eb8a8ae0c).